packages/desktop/components/popups/send/SendConfirmationPopup.svelte

```diff
@@ -70,6 +70,12 @@
     $: isLayer2Transaction = !!layer2Parameters
     $: isTransferring = $selectedAccount.isTransferring
     $: hideGiftToggle = isBaseTokenTransfer || isLayer2Transaction || (disableToggleGift && !giftStorageDeposit)
+    $: initialExpirationDate = getInitialExpirationDate(
```

Comment:
i dont think we can put this here, it breaks the current functionality we have in firefly with the expiration times, a simple example you can see it when trying to send a native token, firefly should set an expirtion time if you dont gift the storage deposit, but with this change, its setting no expiration